cli/internal/api/client.go

```diff
@@ -12,26 +14,23 @@ import (
 const (
 	defaultAPIURL = "https://api.cloudquery.io"
 	envAPIURL     = "CLOUDQUERY_API_URL"
+	envCLIAPIURL  = "CLOUDQUERY_CLI_API_URL"
+	envCLIToken   = "CLOUDQUERY_CLI_TOKEN"
 )
 
+// NewClient creates a new client with the given token.
 func NewClient(token string) (*cloudquery_api.ClientWithResponses, error) {
-	c, err := cloudquery_api.NewClientWithResponses(env.GetEnvOrDefault(envAPIURL, defaultAPIURL),
-		cloudquery_api.WithRequestEditorFn(func(ctx context.Context, req *http.Request) error {
-			req.Header.Set("Authorization", fmt.Sprintf("Bearer %s", token))
-			return nil
-		}))
-	if err != nil {
-		return nil, fmt.Errorf("failed to create api client: %w", err)
-	}
-	return c, nil
+	return newClient(token, false)
 }
 
+// NewAnonymousClient creates a client that doesn't require authentication.
 func NewAnonymousClient() (*cloudquery_api.ClientWithResponses, error) {
-	c, err := cloudquery_api.NewClientWithResponses(env.GetEnvOrDefault(envAPIURL, defaultAPIURL))
-	if err != nil {
-		return nil, fmt.Errorf("failed to create api client: %w", err)
-	}
-	return c, nil
+	return NewClient("")
```

Comment:
Won't this pass `Authorization: Bearer`? Will that work? Won't the service fail the request?

Reply:
No, empty tokens are now considered anonymous (see `opts` init in `newClient()`)

Reply:
Ah ok you changed `NewClient` to call `newClient` 🙃